spec/imsc-hrm.html

```diff
@@ -210,8 +210,9 @@
       removed before advancement to Proposed Recommendation if sufficient implementation experience cannot be
       demonstrated.</p>
 
-      <p>Substantive changes applied since the <a href="https://www.w3.org/TR/2021/WD-imsc-hrm-20211109/">First Public
-      Working Draft</a> are provided for convenience at <a href="#substantive-changes-summary"></a>.
+      <p>Substantive changes applied to the document since its <a
+      href="https://www.w3.org/TR/2021/WD-imsc-hrm-20211109/">First Public Working Draft</a> are informatively listed at
+      <a href="#substantive-changes-summary"></a>.
```

Comment:
Do we want to duplicate the dated version reference both here and in the Summary of substantive changes section? Seems like less maintenance to have it only in one place, which I suggest is not here, but in the Summary of substantive changes section itself.